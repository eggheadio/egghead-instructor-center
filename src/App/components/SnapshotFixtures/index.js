import React from 'react'

export const StringFixture = 'String'

export const ComponentFixture = () => (
  <div>
    Component
  </div>
)
