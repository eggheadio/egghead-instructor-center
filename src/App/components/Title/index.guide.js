import React from 'react'
import {storiesOf} from '@kadira/storybook'
import faker from 'faker'
import Title from '.'

storiesOf('Title', module)
  .add('default', () => (
    <Title>
      {faker.lorem.words()}
    </Title>
  ))
