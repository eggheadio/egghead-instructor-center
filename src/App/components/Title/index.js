import React, {PropTypes} from 'react'

const Title = ({children}) => (
  <header>
    <h1 className='f1'>
      {children}
    </h1>
  </header>
)

Title.propTypes = {
  children: PropTypes.string.isRequired,
}

export default Title
