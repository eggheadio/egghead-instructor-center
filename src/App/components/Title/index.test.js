import React from 'react'
import renderer from 'react-test-renderer'
import {StringFixture} from '../SnapshotFixtures'
import Title from '.'

test('Title', () => {
  const tree = renderer.create(
    <Title>
      {StringFixture}
    </Title>
  )
  expect(tree).toMatchSnapshot()
})
