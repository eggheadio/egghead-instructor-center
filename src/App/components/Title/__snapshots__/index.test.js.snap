exports[`test Title 1`] = `
<header>
  <h1
    className="f1">
    String
  </h1>
</header>
`;
